pkg/hostagent/mount.go

```diff
@@ -41,7 +42,7 @@ func (a *HostAgent) setupMount(ctx context.Context, m limayaml.Mount) (*mount, e
 		return nil, err
 	}
 	// NOTE: allow_other requires "user_allow_other" in /etc/fuse.conf
-	sshfsOptions := "allow_other"
+	sshfsOptions := "allow_other,cache=no"
```

Comment:
This option seems specific to sshfs v2?
https://github.com/libfuse/sshfs/blob/ef3870a99710447a1c9219c6fcfce34bc5ab9760/sshfs.1.in#L67

Doesn't seem supported in sshfs v3 (not documented, at least)
https://github.com/libfuse/sshfs/blob/master/sshfs.rst

Reply:
Ubuntu dpkg has really weird versioning scheme.. not sure whether we are using v3 or v2 🤷‍♂️ 
https://pkgs.org/download/sshfs
`sshfs_3.6.0+repack+really2.10-0ubuntu1_amd64.deb`

Reply:
v3 seems to have undocumented support for `cache=no`, so LGTM

https://github.com/libfuse/sshfs/blob/a2054a2b737577b63e97ea34d8c9976048766719/sshfs.c#L523-L525